tests/testthat/test-warnings.R

```diff
@@ -3,17 +3,14 @@
 test_that("warn_if_vars_exist Test 1: warning if a variable already exists in the input dataset", {
   dm <- pharmaversesdtm::dm
```

Comment:
oohhh can we make a dummy example remove pharmaversesdtm from the Suggest?

Reply:
I think @millerg23 is tackling that in #434